src/clients/repositoryinfoclient.ts

```diff
@@ -82,6 +96,43 @@ export class RepositoryInfoClient {
         return repositoryInfo;
     }
 
+    private splitTfvcCollectionUrl(collectionUrl: string): string[] {
+        let result: string[] = [ , ];
+        if (!collectionUrl) {
+            return result;
+        }
+
+        // Now find the TRUE last separator (before the collection name)
+        let trimmedUrl: string = this.trimTrailingSeparators(collectionUrl);
+        let index: number = trimmedUrl.lastIndexOf("/");
+        if (index >= 0) {
+            // result0 is the server url without the collection name
+            result[0] = trimmedUrl.substring(0, index + 1);
```

Comment:
I have been using slice instead of substring. Let me know which you prefer. I assume they are very similar in how they work on strings.